hazelcast/src/test/java/com/hazelcast/internal/diagnostics/SlowOperationPluginTest.java

```diff
@@ -48,7 +47,6 @@ public void setup() {
         Config config = new Config()
                 .setProperty(SLOW_OPERATION_DETECTOR_ENABLED.getName(), "true")
                 .setProperty(SLOW_OPERATION_DETECTOR_THRESHOLD_MILLIS.getName(), "1000")
-                .setProperty(SLOW_INVOCATION_DETECTOR_THRESHOLD_MILLIS.getName(), "1000")
```

Comment:
This was the last usage of this deprecated `GroupProperty`, so it really had no effect.